### PR TITLE
Google Auth #5

### LINKED ISSUE #5: [Feature] Google auth Firebase
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by vieira-rodrigo**
Performing google authentication.
Fixes #5

### CODE REVIEW

**Review by vcasadei — CHANGES_REQUESTED** [review]
The Google auth works.

However, if I try to create an account with the same Email as I used in the login google auth, the app breaks. Could you check this?

**Comment by vieira-rodrigo** [comment]
> The Google auth works.
> 
> However, if I try to create an account with the same Email as I used in the login google auth, the app breaks. Could you check this?

Done.

File: src/app/pages/login/login.page.ts [hunk 1]
```diff
@@ -8,32 +10,36 @@ import { ToastController, NavController } from '@ionic/angular';
...
-      message: 'TODO - Login Google',
-      duration: 2000
-    });
-    toast.present();
+    this.googleAuth
+    .doAuth()
+    .then(this.goToHome.bind(this))
+    .catch(error =>{
+      console.error(error)
+      this.coronaToast.showError("Could not perform authetication.");
```
Comment (vcasadei) [inline src/app/pages/login/login.page.ts]: Colocar em pt-br

File: src/app/pages/signup/signup.page.ts [hunk 2]
```diff
@@ -49,6 +50,10 @@ export class SignupPage implements OnInit {
         this.navCtrl.setDirection('forward');
         this.navCtrl.navigateForward('/home');
         console.log(this.ionicForm.value);
+      }).catch(error =>{
+        console.error(error)
+        let msg = error.code == 'auth/email-already-in-use' ? 'The email address is already in use' : error.message
```
Comment (vcasadei) [inline src/app/pages/signup/signup.page.ts]: pt-br

**Review by vcasadei — CHANGES_REQUESTED** [review]

**Review by vcasadei — APPROVED** [review]
It works!

**Pull request merged**